test/functional/Progress.hs

```diff
@@ -52,29 +46,10 @@ tests =
                           (command ^. L.command)
                           (decode $ encode $ fromJust $ command ^. L.arguments)
 
-                      expectProgressMessages ["Evaluating"] activeProgressTokens
+                      expectProgressMessages ["Evaluating"] createdProgressTokens activeProgressTokens
                   _ -> error $ "Unexpected response result: " ++ show response
-        , requiresOrmoluPlugin $ testCase "ormolu plugin sends progress notifications" $ do
```

Comment:
So these two tests are not working anymore, because of the way we handle the notification?